sdk/storage/azblob/client.go

```diff
@@ -158,3 +158,7 @@ func (c *Client) DownloadStream(ctx context.Context, containerName string, blobN
 	o = shared.CopyOptions(o)
 	return c.svc.NewContainerClient(containerName).NewBlobClient(blobName).DownloadStream(ctx, o)
 }
+
+func (c *Client) GetUserDelegationCredential(background context.Context, info service.KeyInfo, opts *service.GetUserDelegationKeyOptions) (*service.UserDelegationCredential, error) {
```

Comment:
Is a user delegation credential commonly used, enough that we want it on the tiny client?  I don't know enough about its usage.

Reply:
There are quite a few requests for it, and I think it could be beneficial having it on the tiny client.

Reply:
Could we leave it out for the first beta, until we get some feedback?  We can always add it later but taking it away can be more difficult, and we're really trying to keep the API set on the tiny client as simple as possible.

Reply:
My intention was to address this comment (https://github.com/Azure/azure-sdk-for-go/pull/19141#discussion_r977982704), but I may have misinterpreted this? 

And yes, I can keep it out from the tiny client!

Reply:
Oh I see what you're saying.  Having it a method on the `service.Client` is correct.  Let's just keep it off the tiny client for now until we get some feedback.

Reply:
Sure! Just updated :)